src/main/java/http/request/RequestParser.java

```diff
@@ -0,0 +1,66 @@
+package http.request;
+
+import java.nio.charset.StandardCharsets;
+import java.util.*;
+
+public class RequestParser {
+    public static String startLine;
+    public static HashMap<String, String> headers = new HashMap<>();
+    public static String statusCode;
+    public static String statusMessage;
+    public static byte[] requestBody ;
+
+    public static void setRequestType(String requestData) {
+        System.out.println(requestData);
+        startLine = requestData;
+
+    }
+
+    public static String getRequestType() {
+        System.out.println(startLine);
+        return startLine;
+    }
+
+    public static void setHeaders( String value) {
+        headers.put("Content-Type", value);
```

Comment:
I'm interested in what's happening here. Presumably you might want to send other headers back on a response as well as Content-Type? Can you think of a way that this method could be made more flexible?

Reply:
Yes I was hardcoding it to get the test to pass not sure how i'll abstract it but I'll try.